source/prefab/Card.h

```diff
@@ -0,0 +1,83 @@
+#ifndef EMP_CARD_INTERFACE_H
+#define EMP_CARD_INTERFACE_H
+
+#include "../web/Div.h"
+#include "../tools/string_utils.h"
+#include "Collapse.h"
+#include "FontAwesomeIcon.h"
+
+namespace emp {
+    namespace prefab{
+        class Card : public web::Div{
+            private:
+                web::Div collapse_link;
+                web::Div collapse_body;
+                std::string card_base = this->GetID();
+                web::Div card_header{emp::to_string(card_base, "_card_header")};
+                web::Div card_body{emp::to_string(card_base, "_card_body")};
+                void AddBootstrap(){
+                    this->SetAttr("class", "card");
+                    card_header.SetAttr("class", "card-header");
+                    card_body.SetAttr("class", "card-body");
+                    // card_header.AddClass("collapse_toggle_card_header");
+                }
+            public:
+                enum Collapse {NONE, OPEN, CLOSED}; // collapse options for constructing a card 
+                Card(Collapse state=NONE, bool showGlyphs=true, const std::string & in_name=""): web::Div(in_name){
+                    AddBootstrap();
+                    if(state != NONE){ // if card is collapsible, make the collapse link the head of the card
+                        if(state == OPEN){ 
+                            prefab::Collapse accordion(card_header, card_body, true, emp::to_string(card_base+ "_card_collapse"));
+                            collapse_link = accordion.GetLinkDiv();
+                            collapse_body = accordion.GetToggleDiv();
+                        }
+                        else{
+                            prefab::Collapse accordion(card_header, card_body, false, emp::to_string(card_base + "_card_collapse"));
+                            collapse_link = accordion.GetLinkDiv();
+                            collapse_body = accordion.GetToggleDiv();
+                        }
+                        if(showGlyphs){ // by default add glyphs to a collapsible card
+                        // TODO: Don't use float, just set it to align to the right (need to make icon decend from div first)
```

Comment:
I added this todo because if header content was not added to a card that had toggle icons, the header size would be smaller than the icon size. However, when I tested it again, this didn't happen. I think we don't actually need to do this.